### PR TITLE
wh: pendingEvents: handle null values while querying for max(end_stag…

### PR DESCRIPTION
**Pull request by dhawal1248**
…ing_file_id)

## Description of the change

> Description here

## Notion Link

> Notion Link

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: warehouse/warehouse.go [hunk 1]
```diff
@@ -1408,14 +1408,18 @@ func getPendingStagingFileCount(sourceOrDestId string, isSourceId bool) (fileCou
...
-	err = dbHandle.QueryRow(sqlStatement).Scan(&lastStagingFileID)
+	err = dbHandle.QueryRow(sqlStatement).Scan(&lastStagingFileIDRes)
 	if err != nil && err != sql.ErrNoRows {
 		err = fmt.Errorf("Query: %s failed with Error : %w", sqlStatement, err)
 		return
 	}
+	lastStagingFileID := int64(0)
+	if lastStagingFileIDRes.Valid {
+		lastStagingFileID = lastStagingFileIDRes.Int64
+	}
```
Comment (abhimanyubabbar) [inline warehouse/warehouse.go:1422]:
```suggestion
	lastStagingFileID := lastStagingFileIDRes.Int64
```

This should work as well right given the default value is zero ?

**Review by achettyiitr — APPROVED** [review]
LGTM

**Review by Jayachand — APPROVED** [review]
LGTM

File: warehouse/warehouse.go (hunk 1 above)
Comment (abhimanyubabbar) [reply warehouse/warehouse.go:1422]: Sounds good. It's not a blocker from my end, just a suggestion !

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by Jayachand — APPROVED** [review]

**Pull request merged**